hs-bindgen/fixtures/program_slicing.pp.hs

```diff
@@ -0,0 +1,38 @@
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE NoImplicitPrelude #-}
+{-# LANGUAGE StandaloneDeriving #-}
+
+module Example where
+
+import qualified Foreign as F
+import qualified HsBindgen.Runtime.Prelude
+import Prelude ((<*>), (>>), Eq, Int, Show, pure)
+
+data Foo = Foo
+  { foo_sixty_four :: HsBindgen.Runtime.Prelude.Word64
```

Comment:
Somehow, it prefers the default descriptive binding specs, which was not the case a few days ago. (I guess this is because of changes Travis has made in the meantime).

Reply:
Fixed by disabling descriptive binding specifications in this test. I also adjusted the nonSelectedDecls, but I am unsure whether this is actually required.

Reply:
Also, I noticed that we still call those _external_ bindings specifications instead of _descriptive_. Is this on somebody's radar?